platforms/vercel-with-redwood/api/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "@prisma/client": "2.21.0-dev.17",
-    "@redwoodjs/api": "0.28.4",
+    "@redwoodjs/api": "0.29.0",
```

Comment:
```suggestion
    "@redwoodjs/api": "0.29.0",
    "@redwoodjs/api-server": "0.29.0",
```

In Redwood v0.28.1, we added the api-server package as another requirement. This might be causing the failure here.
- See Code Modifications section of the [v0.29.0 Release Notes](https://github.com/redwoodjs/redwood/releases/tag/v0.29.0)

Reply:
^^ @janpio

Reply:
For reference, here are the current respective package.json files included at time of installation:
- [./package.json](https://github.com/redwoodjs/redwood/blob/main/packages/create-redwood-app/template/package.json)
- [api/package.json](https://github.com/redwoodjs/redwood/blob/main/packages/create-redwood-app/template/api/package.json)
- [web/package.json](https://github.com/redwoodjs/redwood/blob/main/packages/create-redwood-app/template/web/package.json)

Reply:
Let's see what CI says.

Reply:
All green, so this will be auto merged soon.
Thanks for the quick help!